services/search-repository.js

```diff
@@ -0,0 +1,66 @@
+const Search = require("../models/Search");
+
+addSearch = async ({ user, game, limit }) => {

```

Comment:
Please add `const` before the variable name since this is an initialization (e.g. `const myVariableName = async () => {}`).

You can also opt to just go with the traditional function declaration, e.g.:
```javascript
function myFunctionName () {
}
```

Using either in this use case should be ok since you're not using `this` inside your method.

Reply:
Please apply this to other methods in this file as well.

Reply:
Addressed in fdb073c1ed2d68842125314cf82163988b97a481